casttube/YouTubeSession.py

```diff
@@ -132,7 +134,7 @@ def _initialize_queue(self, video_id, list_id=""):
         """
         request_data = {LIST_ID: list_id,
                         ACTION: ACTION_SET_PLAYLIST,
-                        CURRENT_TIME: "0",
+                        CURRENT_TIME: -1,
```

Comment:
I did some testing here and it seems to me that it didn't affect actual YouTube and causes YouTube TV to go to the "Live" section of the video instead of the current recording. As long as we still have the ability to pass `CURRENT_TIME` as `-1` into YouTube TV then this being a variable won't be a problem, s long as we make it default to `-1` for YouTube TV.